google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/ITQueryTest.java

```diff
@@ -423,6 +424,27 @@ public void bindDateNull() {
     assertThat(row.isNull(0)).isTrue();
   }
 
+  @Test
+  public void bindUuid() {
+    assumeFalse("Emulator does not support UUID yet", isUsingEmulator());
+    assumeTrue("UUID is currently only supported in cloud-devel", isUsingCloudDevel());
```

Comment:
Since same statements are used multiple places, can these statements declared as global variable in the class  ?

Reply:
They are temporary and used for UUID. Hence, not making it at a class level.